R/centrality.R

```diff
@@ -952,9 +952,8 @@ eigen_defaults <- function() {
 #' @param graph Graph to be analyzed.
 #' @param directed Logical scalar, whether to consider direction of the edges
 #'   in directed graphs. It is ignored for undirected graphs.
-#' @param scale Logical scalar, whether to scale the result to have a maximum
-#'   score of one. If no scaling is used then the result vector has unit length
-#'   in the Euclidean norm.
+#' @param scale `lifecycle::badge("deprecated")` Normalization will always take
+#' place.
```

Comment:
There is more than one way to normalize, so let's not simply call this normalization.

```suggestion
#' @param scale `lifecycle::badge("deprecated")` The result is always scaled to have
#' a maximum score of one.
```

I only see the comment "If no scaling is used then the result vector has unit length in the Euclidean norm." This wasn't actually true, so it's good to remove it ASAP.